src/components/option-group/option-group.tsx

```diff
@@ -1,7 +1,7 @@
 import { Component, h, Prop, VNode, Watch, Event, EventEmitter, Fragment } from "@stencil/core";
 
 /**
- * @slot - A slot for adding `calcite-option`s.
+ * @slot - A slot for adding `calcite-option` elements.
```

Comment:
Should we refer to `calcite-option` as an element, custom element, or component? I wonder what the best wording would be for referencing other components. Is it as elements? In properties, we reference the current element as a component.

Reply:
To be consistent with other text, maybe just `calcite-option`s.

Reply:
Was going back and forth on this one, your recommendation looks best. Going to add this in to our consistency docs as well. 😃